DOL.WHD.Section14c.Web/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html ng-app="14c" lang="en">
 <head>
+    <meta charset="utf-8">
```

Comment:
Needs another `meta` tag for IE:

```html
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
```

[Because IE behaves sub-optimally](https://support.microsoft.com/en-us/help/928847/internet-explorer-uses-the-wrong-character-set-when-it-renders-an-html)